terraform/envs/integration/backend.tf

```diff
@@ -0,0 +1,6 @@
+terraform {
+  backend "gcs" {
+    prefix = "app-qanightlyresults"
+    bucket = "terraform-testing-migration"
```

Comment:
There is no bucket for terraform, I created one (manually). It has to be deleted or created with Terraform. (We have to change project for this repository anyway)